docs/en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -213,50 +213,139 @@ This is not mandatory unless the relationship would be otherwise ambiguous.
+[api:SilverStripe\ORM\ManyManyThroughList] rather than the object.
+
+	:::php
+	$team = Team::get()->byId(1);
+
+	$supporters = $team->Supporters();
+	// returns a 'ManyManyList' instance.
+
+
+### Automatic many_many table
+
+If you specify only a single class as the other side of the many-many relationship, then a
+table will be automatically created between the two (this-class)_(relationship-name), will
+be created with a pair of ID fields.
+
+Extra fields on the mapping table can be created by declaring a `many_many_extraFields`
+config to add extra columns.
+
+
 	:::php
 	<?php
 
 	class Team extends DataObject {
-
-	  private static $many_many = array(
+	  private static $many_many = [
 	    "Supporters" => "Supporter",
-	  );
+	  ];
+	  private static $many_many_extraFields = [
+	    'Supporters' => [
+	      'Ranking' => 'Int' 
+	    ]
+	  ];
 	}
 
 	class Supporter extends DataObject {
 
-	  private static $belongs_many_many = array(
+	  private static $belongs_many_many = [
 	    "Supports" => "Team",
-	  );
+	  ];
+	}
+
+
+### many_many through relationship joined on a separate DataObject
+
+If necessary, a third DataObject class can instead be specified as the joining table,
+rather than having the ORM generate an automatically scaffolded table. This has the following
+advantages:
+
+ - Allows versioning of the mapping table, including support for the
+   [ownership api](/developer_guides/model/versioning).
+ - Allows support of other extensions on the mapping table (e.g. subsites).
+ - Extra fields can be managed separately to the joined dataobject, even via a separate
+   GridField or form.
+
+This is declared via array syntax, with the following keys on the many_many:
+ - `through` Class name of the mapping table
+ - `from` Name of the has_one relationship pointing back at the object declaring many_many
+ - `to` Name of the has_one relationship pointing to the object declaring belongs_many_many.
+
+The syntax for `belongs_many_many` is unchanged.
+
+	:::php
+	<?php
+
+	class Team extends DataObject {
+	  private static $many_many = [
+	    "Supporters" => [
+	      'through' => 'TeamSupporter',
+	      'from' => 'Team',
+	      'to' => 'Supporter',
+	    ]
+	  ];
 	}
 
-Much like the `has_one` relationship, `many_many` can be navigated through the `ORM` as well. The only difference being
-you will get an instance of [api:ManyManyList] rather than the object.
+	class Supporter extends DataObject {
+	  private static $belongs_many_many = [
+	    "Supports" => "Team",
+	  ];
+	}
+	
+	class TeamSupporter extends DataObject {
+	  private static $db = [
+	    'Ranking' => 'Int',
+	  ];
+	  
+	  private static $has_one = [
+	    'Team' => 'Team',
+	    'Supporter' => 'Supporter'
+	  ];
+	}
+
+In order to filter on the join table during queries, you can use the "Join" table alias
+for any sql conditions.
+
 
 	:::php
 	$team = Team::get()->byId(1);
+	$supporters = $team->Supporters()->where(['"Join"."Ranking"' => 1]);
```

Comment:
Are we able to use the join table name directly here? To use the physician/patient example from [Ruby on Rails](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association):

```php
$patientsToday = $physician->Patients()->where(['"Appointment"."Date"' => date('Y-m-d')]);
```

Is that possible, or do we have to use the `Join` alias?